app/screens/home-screen/home-screen.tsx

```diff
@@ -332,6 +355,11 @@ export const HomeScreen: React.FC = () => {
           </>
         ) : null}
         <AppUpdate />
+        <SetDefaultAccountModal
+          isVisible={setDefaultAccountModalVisible}
+          toggleModal={toggleSetDefaultAccountModal}
+          onSelectedAccount={onSelectedAccount}
```

Comment:
what do we need to pass onSelectedAccount from `home` to the modal? 

the logic feel convoluted.

Reply:
Its essentially a callback on completion of setting the account. If we put the logic to navigate to the receive screen after completion, it makes it so that we can not reuse the modal in other places.

Reply:
> Its essentially a callback on completion of setting the account. If we put the logic to navigate to the receive screen after completion, it makes it so that we can not reuse the modal in other places.

I understand the goal but unless a modal is used in multiple place I'm not sure this is useful to get this callback as it make the code more spread around for a maybe future use case.

I guess it's just a matter of perspective.

Reply:
I see your point. I can remove it for now until we need to reuse the component.